ferrilata-devilline/Services/MockPitchService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using ferrilata_devilline.Models;
+using ferrilata_devilline.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ferrilata_devilline.Services
+{
+    public class MockPitchService : IPitchService
+    {
+        public Pitches GetPitches()
+        {
+            return GeneratePitches();
+        }
+
+        private Pitches GeneratePitches()
```

Comment:
It looks like we have two methods for one task of getting Pitches - can be simplified to one